lib/autoproj/daemon/git_api/services/github.rb

```diff
@@ -30,32 +30,145 @@ class GitHub < GitAPI::Service
 
                     NUMBER_RX = /\#(\d+)/.freeze
 
-                    def initialize(**options)
-                        super
+                    # @return [Symbol] "auto", the merge commit used in the CI
+                    #   build will be the GitHub-generated merge commit if
+                    #   available, and the HEAD of the pull request's branch
+                    #   otherwise. If "merge", it is always the merge commit
+                    #   unless the pull request is a draft.  If "head", it is
+                    #   always the pull request's branch's HEAD.
+                    attr_accessor :pr_commit_strategy
 
+                    # @param [Number] mergeability_timeout how long, in seconds,
+                    #   we wait for GitHub to compute a pull request's mergeability
+                    # @param [Symbol] pr_commit_strategy If "auto", the merge
+                    #   commit used in the CI build will be the GitHub-generated
+                    #   merge commit if available, and the HEAD of the pull
+                    #   request's branch otherwise. If "merge", it is always
+                    #   the merge commit unless the pull request is a draft.
+                    #   If "head", it is always the pull request's branch's
+                    #   HEAD.
+                    def initialize(
+                        pr_commit_strategy: "auto", mergeability_timeout: 60,
+                        **options
+                    )
+                        super(**options)
+
+                        @client = create_octokit_client(cache: false)
```

Comment:
`cache: true` ?

Reply:
Yes. Did not run the tests before I pushed :shame: